docs/UserGuide.adoc

```diff
@@ -445,25 +461,10 @@ Examples:
 * `gradebook assign c/CS2040C i/A0168111 g/Assignment 1 m/10` +
 Assigns 10 marks to CS2040C student with matric number A0168111.
 
-=== Selecting a person : `select`
-
-Selects the person identified by the index number used in the displayed person list. +
-Format: `select INDEX`
-
-****
-* Selects the person and loads the Google search page the person at the specified `INDEX`.
-* The index refers to the index number shown in the displayed person list.
-* The index *must be a positive integer* `1, 2, 3, ...`
-****
-
-Examples:
+==== Viewing help : `help`
 
-* `list` +
-`select 2` +
-Selects the 2nd person in the address book.
-* `find Betsy` +
-`select 1` +
-Selects the 1st person in the results of the `find` command.
+Shows the help page +
+Format: `help`
```

Comment:
Viewing help is nested under Grade Management as 3.4.7. Might wanna bring it up one level :)